lib/cli/src/frameworks/ember/stories/Header.stories.js

```diff
@@ -0,0 +1,30 @@
+import { hbs } from 'ember-cli-htmlbars';
+// You don't need the css import in your own components
+import './header.css';
+
+export default {
+  title: 'Example/Header',
+  argTypes: {
+    onLogin: { action: 'onLogin' },
+    onLogout: { action: 'onLogout' },
+    onCreateAccount: { action: 'onCreateAccount' },
+  },
+};
+
+const Template = (args) => ({
+  template: hbs`{{storybook-header
+    user=user
+    onLogin=(action onLogin)
+    onLogout=(action onLogout)
+    onCreateAccount=(action onCreateAccount)
+  }}`,
```

Comment:
```suggestion
  template: hbs`<StorybookHeader
    @user={{this.user}}
    @onLogin={{action this.onLogin}}
    @onLogout={{action this.onLogout}}
    @onCreateAccount={{action this.onCreateAccount}}
  />`,
```

See https://guides.emberjs.com/release/upgrading/current-edition/templates/#toc_angle-bracket-syntax for more context :)